src/sentry/features/base.py

```diff
@@ -1,24 +1,26 @@
 __all__ = ["Feature", "OrganizationFeature", "ProjectFeature", "ProjectPluginFeature"]
 
+from typing import Any
+
 
 class Feature:
-    def __init__(self, name):
+    def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
```

Comment:
Why does this need `*args` and `**kwargs`?

Reply:
I left a note in the constructor's docstring. In `FeatureManager.get()` we do:
```
cls = self._get_feature_class(name)
return cls(name, *args, **kwargs)
```
to handle passing in `organization`, `project`, and `plugin`.